cmd/deviceinfo_linux.go

```diff
@@ -1,4 +1,4 @@
-package main
```

Comment:
Let's name this device_info_linux.go

Reply:
Oh, and I realized that some of the files are missing the copyright header. Don't forget to add those and put a RedHat 2025 line, as well.